protos/telemetry/telemetry.proto

```diff
@@ -575,9 +575,12 @@ enum FixType {
 
 // Battery type.
 message Battery {
-    uint32 id = 3 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
-    float voltage_v = 1 [(mavsdk.options.default_value)="NaN"]; // Voltage in volts
-    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0.0 to 1.0)
+    uint32 id = 1 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
+    float temperature_cdegc = 2 [(mavsdk.options.default_value)="NaN"]; // Temperature of the battery in centidegrees Celsius. NAN for unknown temperature
```

Comment:
Why centidegrees? Sorry to bring units up again but that's a bit weird isn't it? When it's a uint type, then that sort of thing makes sense but not for a float, in my opinion.

Reply:
Agree. It is to match what have been done with the [battery v2](https://github.com/mavlink/mavlink/pull/1846/files#diff-413c1cf6a533671c65dda0eeaa0280a129cbf251a06d3ad7ef9ea2bbdb34f1f6R417-R424) but if we agree to have a mismatch I also prefer to go with degree

Reply:
Ok, I'd say please do.

Reply:
Done and ready to be merged :)